code/modules/mob/transform_procs.dm

```diff
@@ -81,7 +81,7 @@
 
 	if(move)
 		var/obj/new_location = null
-		for(var/turf/sloc in getSpawnLocations(/datum/job/ai))
+		for(var/turf/sloc in getSpawnLocations("AI"))
```

Comment:
Paths is better than text strings.

Reply:
Some places use name for output

Reply:
But jobs use paths.
Even AI job few lines after that.

Reply:
not anymore, below was an error

Reply:
There is an example, "Cryogenic Storage" is a name for other spawnpoint. Name is used for output and since they share the same list it will be confusing. And also previously other jobs used names not types, observer and ai were exceptions

Reply:
That seems it will not work propertly.

Reply:
It can't use paths and text simultaniously for same spawnpoint

Reply:
it was completly reworked it works